paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/addresselement/AddressElementExampleActivity.kt

```diff
@@ -0,0 +1,112 @@
+package com.stripe.android.paymentsheet.example.samples.ui.addresselement
+
+import android.os.Bundle
+import androidx.activity.compose.setContent
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Button
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import com.stripe.android.paymentsheet.addresselement.AddressDetails
+import com.stripe.android.paymentsheet.addresselement.AddressLauncher
+import com.stripe.android.paymentsheet.addresselement.rememberAddressLauncher
+import com.stripe.android.paymentsheet.example.R
+
+class AddressElementExampleActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        supportActionBar?.title = getString(R.string.address_element_title)
+
+        setContent {
+            val viewModel by viewModels<AddressElementExampleViewModel>()
+            val viewState by viewModel.state.collectAsState()
+
+            val addressLauncher = rememberAddressLauncher(
+                callback = viewModel::handleResult,
+            )
+
+            Column(
+                verticalArrangement = Arrangement.Center,
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier.fillMaxSize(),
+            ) {
+                when (val state = viewState) {
+                    is AddressElementExampleViewState.Loading -> {
+                        CircularProgressIndicator()
+                    }
+                    is AddressElementExampleViewState.Content -> {
+                        state.address?.let { address ->
+                            Address(address)
+                        }
+
+                        Button(
+                            onClick = {
+                                val config = AddressLauncher.Configuration.Builder()
+                                    // Provide your Google Places API key to enable autocomplete
+                                    .googlePlacesApiKey(null)
```

Comment:
Should we use metadata to inject the google places API key here, similar to how we do it in the playground activity? Maybe can do as a follow up.